exercises/practice/diffie-hellman/.docs/instructions.md

```diff
@@ -19,7 +19,7 @@ the same to pick a private key b.
 
 Alice calculates a public key A.
 
-    A = g**a mod p
+    A = gᵃ mod p
```

Comment:
Interesting, i didn't realize that Unicode or markdown would handle this.

Reply:
There was **quite** the discussion on that one!  😄   [diffie-hellman](https://github.com/exercism/problem-specifications/pull/1933).